src/fuzzer.rs

```diff
@@ -101,7 +101,7 @@ pub fn fuzz() -> Result<()> {
         code_coverage_feedback,
         time_feedback, // Time feedback, this one does not need a feedback state
     );
-    let state = init_state(&mut objective, &mut collective_feedback, &mut state_uninit)?;
+    state.initialize(&mut objective, &mut collective_feedback)?;
```

Comment:
We could consider returning the state as a reference such that we can stick to the `state_uninit` and `state` distinction